apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -142,15 +142,15 @@ const BookingPage = (props: BookingPageProps) => {
     [LocationType.Huddle01]: "Huddle01 Video",
     [LocationType.Tandem]: "Tandem Video",
   };
-
+  const loggedInIsOwner = eventType.users[0].name === session?.user.name;
```

Comment:
A little util to check if the user name and the event owner name's are the same, to avoid pre-filling your own bookings